### PR TITLE
Adding reminder for incidents that have been stable for over a week.

### PR DESCRIPTION
**Pull request by kevgliss**
Reminds every week on Monday, I chose not to make this configurable at this time.

### CODE REVIEW

File: src/dispatch/incident/messaging.py [hunk 1]
```diff
@@ -591,3 +593,32 @@ def send_incident_resources_ephemeral_message_to_participant(
...
     log.debug(f"List of incident resources sent to {user_id} via ephemeral message.")
+
+
+def send_incident_close_reminder(incident: Incident):
+    """Sends a direct message to the incident reminding them to close the incident if possible."""
+    message_text = f"Incident Status Reminder"
+    message_template = INCIDENT_STATUS_REMINDER
+
+    convo_plugin = plugins.get(INCIDENT_PLUGIN_CONVERSATION_SLUG)
+    update_command = convo_plugin.get_command_name(ConversationCommands.edit_incident)
```
Comment (mvilanova) [inline src/dispatch/incident/messaging.py:604]: nit: we should change the name of the enum to `update_incident` to be consistent with the command name.

File: src/dispatch/incident/scheduled.py [hunk 2]
```diff
@@ -289,3 +288,17 @@ def calculate_incidents_cost(db_session=None):
             # we shouldn't fail to update all incidents when one fails
             log.error(e)
             sentry_sdk.capture_exception(e)
+
+
+@scheduler.add(every(1).day.at("18:00"), name="incident-stable-reminder")
```
Comment (mvilanova) [inline src/dispatch/incident/scheduled.py:293]: `incident-close-reminder` or `incident-status-reminder`?

File: src/dispatch/incident/messaging.py (hunk 1 above)
Comment (kevgliss) [reply src/dispatch/incident/messaging.py:604]: Yeah, I had `update_incident` first until I realized that wasn't the enum.

Comment (mvilanova) [reply src/dispatch/incident/messaging.py]: Yeah, looks like we never updated the enum when we renamed the command from edit to update.

Comment (mvilanova) [reply src/dispatch/incident/messaging.py]: Don't forget to update it here as well: https://github.com/Netflix/dispatch/blob/develop/src/dispatch/conversation/enums.py#L6

**Pull request merged**